ionizer/ops.py

```diff
@@ -109,10 +114,16 @@ def compute_matrix(phi):  # pylint: disable=arguments-differ
         array([[ 0.70710678+0.j        , -0.33900505-0.62054458j],
                [ 0.33900505-0.62054458j,  0.70710678+0.j        ]])
         """
-        exponent = -1j * phi
+        if qml.math.get_interface(phi) == "tensorflow":
+            phi = qml.math.cast_like(phi, 1j)
+
+        exponent = (0 -1j) * phi
+        a = (1 + 0j) / qml.math.sqrt(2)
+        b = ((0 - 1j) * qml.math.exp(exponent)) / qml.math.sqrt(2)
+        c = ((0 - 1j) * qml.math.exp(qml.math.conj(exponent))) / qml.math.sqrt(2)
```

Comment:
```suggestion
        c = qml.math.conj(b)
```
Is this correct? If so, can just use `conj` directly in the return statement rather than creating a separate variable for it.